api/users.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt')
+const { JWT_SECRET } = process.env;
+const { createUser, getUser, getUserByUsername } = require('../db');
+const { requireUser } = require('./utils');
+
+
+// users/
+router.get('/', async (req, res, next) => {
+    const users = await getUser();
+    try {
+        res.send({
+            users
+        })
+    } catch (error) {
+        next(error)
+    }
+});
```

Comment:
our current getUser function doesn't grab all users, it grabs specifically one user using the username and password,

we can write a getAllUsers function if we'd like and call that instead, or edit our getUser function to work differently.

another thing to consider is that this api route should only be accessible by an administrator, so we will likely need to write a helper function, maybe something like requireAdminUser to call on this route

for consistency, i would change this catch to also be 
`catch ({ name, message }) {
    next({ name, message })
}`